### PR TITLE
Close timed-out games

### LINKED ISSUE #49: On NewGame, if there is an open game which has timed out, close it

### PR DESCRIPTION
**Pull request by ajnavarro**
Uploading this trying to fix #49

### CODE REVIEW

File: realm/chess.gno [hunk 1]
```diff
@@ -450,7 +451,7 @@ func ClaimTimeout(gameID string) string {
 			g.Concluder = &g.White
 		}
 		g.Winner = WinnerNone
-		return g.json()
+		return g.json(), nil
```
Comment (zivkovicmilos) [inline realm/chess.gno:454]: @thehowl 
What implications does this have for the return type parsing?

Comment (moul) [reply realm/chess.gno:454]: the function is now unexported.

but anyway, it should just join the return values (with type), with a comma.

Comment (thehowl) [reply realm/chess.gno:454]: @zivkovicmilos None, Antonio created a clone without an error which is exported and should work as normal

File: realm/chess.gno [hunk 2]
```diff
@@ -229,6 +227,11 @@ func newGame(caller, opponent std.Address, seconds, increment int) *Game {
 	var games []*Game
 	if exist {
 		games, _ = val.([]*Game)
+
+		// Close previous games that timed-out
+		for _, g := range games {
+			_, _ = claimTimeout(g.ID)
```
Comment (thehowl) [inline realm/chess.gno:233]: better if we can have the unexported `claimTimeout` accept directly a `*Game` (this way we don't need to re-fetch Game again from gameStore, seeing as they are the same pointer it is not necessary

**Review by zivkovicmilos — APPROVED** [review]

**Pull request merged**